src/backends/fixerio/index.js

```diff
@@ -60,26 +51,15 @@ class FixerIoBackend {
       })
       apiData = result.body
     }
-
+    const pairs = config.get('tradingPairs')
```

Comment:
Yeah, it's possible this is empty, which is why I had to add a check here: https://github.com/interledger/five-bells-connector/pull/124/files#diff-9439739e49d60f21e447a7bdcc0ef546R7. I can do the same here.